src/uniprotkb/components/entry/ProteinProcessingSection.tsx

```diff
@@ -24,7 +29,18 @@ const ProteinProcessingSection = ({
   sequence,
   primaryAccession,
 }: Props) => {
-  if (!hasContent(data)) {
+  const { loading: proteomicsPtmLoading, data: proteomicsPtmData } =
+    useDataApi<ProteomicsPtm>(proteinsApi.proteomicsPtm(primaryAccession));
+
+  if (proteomicsPtmLoading) {
+    return <Loader />;
+  }
+
+  const convertedPtmExchangeFeatures = proteomicsPtmData
+    ? convertPtmExchangeFeatures(proteomicsPtmData.features)
+    : [];
+
+  if (!hasContent(data) && !convertedPtmExchangeFeatures.length) {
     return null;
   }
```

Comment:
Is this what triggers the side-panel to decide to render the shortcut to the section? If this is it it might be good to keep that in mind to have the same approach for the structure and the variation sections (not part of this PR though)

Reply:
Note that the variations endpoint can be heavy, and this one might also be heavy too in the future. But seeing that these endpoints return 404 when no data, a quick and cheap way to check might be to do a "HEAD" query and see the status code, and do the actual data loading within a lazy component (like for the variations endpoint)